addons/point_of_sale/static/src/app/generic_components/list_container/list_container.js

```diff
@@ -63,11 +74,6 @@ export class ListContainer extends Component {
             this.popover.close();
             return;
         }
-        // the popover will position itself underneath the specified div,
-        // but in this case we want it to be right on top of it;
-        // in order to do this we set the height of our div to 0 for the
-        // duration that the popover is shown.
-        this.container.el.style.height = 0;
```

Comment:
this logic was moved in the template in an effort to make the code more declarative